src/components/NegotiationForm.vue

```diff
@@ -146,6 +146,12 @@
         </div>
       </tab-content>
       <template #footer="props">
+        <div v-if="props.isLastStep" class="form-check mb-3 d-flex justify-content-end">
+          <input class="form-check-input" type="checkbox" value="" id="flexCheckDefault">
+          <label class="form-check-label ps-2" for="flexCheckDefault">
+             <p class="m-0">Start Test Negotiation</p> 
```

Comment:
Please change to "Mark as test request" and would it be possible to display it only if the user has role admin?

Reply:
changed text to  "Mark as test request", yes, it is only visible if role is Admin